lib/Models/UserDrawing.ts

```diff
@@ -483,9 +596,17 @@ export default class UserDrawing extends MappableMixin(
             hierarchy: new CallbackProperty(function () {
               return new PolygonHierarchy(that.getPointsForShape());
             }, false),
-            material: new Color(0.0, 0.666, 0.843, 0.25),
+            //material: new Color(0.0, 0.666, 0.843, 0.25),
+            material: this.autoClosePolygon
+              ? new Color(0.0, 0.666, 0.843, 0.25)
+              : new Color(0.0, 0.666, 0.843, 0),
             outlineColor: new Color(1.0, 1.0, 1.0, 1.0),
-            perPositionHeight: true as any
+            //perPositionHeight: true as any
+            // Clamp to ground polygons of Measure Tool
+            heightReference: new ConstantProperty(
+              HeightReference.CLAMP_TO_GROUND
+            ),
+            perPositionHeight: <any>false
```

Comment:
Are the commented out lines of code leftovers from debugging?

The previous code for `perPositionHeight` is `true as any`, does `false as any` not work?